cql3/sets.hh

```diff
@@ -61,9 +61,10 @@ public:
     class value : public terminal, collection_terminal {
     public:
         std::set<managed_bytes, serialized_compare> _elements;
+        data_type _my_type;
```

Comment:
Done.
At first it is virtual, because this allows to nicely divide implementation for different types into commits. Also I didn't want to rewrite it. Then one commit changes it to a non-virtual function.